### PR TITLE
fix: Improve AT experience in color contrast dialog

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Details

A recent internal bug pointed out that our tab stops were inconsistent in CCA if a user disabled then enabled auto-detection. Digging deeper, we had lots of issues in CCA, including:

- When navigating via keyboard, there were several places where we have no focus indicator
- In the description for auto-detection of colors, "evaluate" was misspelled
- Hiding the contrast for an unknown contrast did nothing to remove the associated label, which is confusing
- Help text for graphical objects was wrong (was the help text for small objects)
- We used "regular objects" and "small objects for the same thing and sometimes used different text on screen or for the AT

This PR fixes all of these issues. Also, @jalkire and I discussed how the initial focus in other places is the instructions, so I did the same here (previously the initial focus was the help link).

##### Motivation

Addresses [1858779}(https://mseng.visualstudio.com/DefaultCollection/1ES/_workitems/edit/1858779), improves keyboard-only experience

##### Video with changes
This video shows tabbing around the CCA dialog. It also shows the NVDA speech viewer to reflect what is exposed to the AT
cca-tabstops

#### Pull request checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/main/docs/Scenarios.md) completed?
- [x] Does this address an existing issue? If yes, Issue - [1858779}(https://mseng.visualstudio.com/DefaultCollection/1ES/_workitems/edit/1858779)
- [x] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [x] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml [hunk 1]
```diff
@@ -104,18 +104,17 @@
...
-                               AutomationProperties.LiveSetting="Polite" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=PrimaryFGBrush}"
-                               AutomationProperties.Name="{Binding Text, RelativeSource={RelativeSource Self}, StringFormat={x:Static Properties:Resources.ColorContrast_OutputAutomationName}}"
-                               KeyboardNavigation.TabIndex="3" Focusable="True"
-                               Text="{Binding Path=ContrastVM.FormattedRatio, RelativeSource={RelativeSource Mode=FindAncestor,AncestorType=local:ColorContrast}}">
-                                </TextBlock>
+                                 AutomationProperties.LiveSetting="Polite" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=PrimaryFGBrush}"
+                                 AutomationProperties.Name="{Binding Text, RelativeSource={RelativeSource Self}, StringFormat={x:Static Properties:Resources.ColorContrast_OutputAutomationName}}"
+                                 Focusable="True" Style="{StaticResource TbFocusable}"
```
Comment (DaveTryon) [inline src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml:109]: Added tbFocusable style, removed KeyboardNavigation.TabIndex. Sort of hard to tell due to white space changes

File: src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml [hunk 2]
```diff
@@ -233,7 +233,7 @@
                             </fabric:FabricIconControl.Style>
                         </fabric:FabricIconControl>
                         <TextBlock x:Name="largeTextResult" KeyboardNavigation.IsTabStop="True" Focusable="True" FontWeight="SemiBold" FontSize="{DynamicResource ConstStandardTextSize}" Margin="8 0 0 0" 
-                               AutomationProperties.HelpText="{x:Static Properties:Resources.largeTextResultAutomationHelpText}" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=PrimaryFGBrush}">
+                               AutomationProperties.HelpText="{x:Static Properties:Resources.ColorContrast_LargeTextResultHelpText}" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=PrimaryFGBrush}">
```
Comment (DaveTryon) [inline src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml:236]: This was just a rename of the resource to be consistent with other resource names

File: src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs [hunk 3]
```diff
@@ -279,14 +286,7 @@ private void HandleToggleButtonSwitch(bool isEnabled)
                 CCAMode.HandleToggleStatusChanged(isEnabled);
             }
 
-            spConfidence.Children.Clear();
```
Comment (DaveTryon) [inline src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs:282]: It's easier to just hide the confidence controls than to rebuild the children each time

**Review by jalkire — APPROVED** [review]
👍

**Pull request merged**